static/js/formula.js

```diff
@@ -221,10 +221,17 @@ function is_valid_formula(str) {
 
     var isValid = /\d*[A-Z]{1}[a-z]?\d*/.test(str).valueOf();
     var usedElements = [];
-    var form_qty = front_number(str);
+    var formulaString = str;
+    var qty = 0;
+
+    /* Gets rid of the leading number for validation*/
+    if(/^\d+/.test(str)){
+        qty = str.match(/^\d+/)[0];
+        formulaString = str.substring(qty.length);
+    }
```

Comment:
Before, formulas with leading numbers were rejected because the length of the elements and their quantities did not match up with the length of the original string. This is because the formula's quantity was never actually included in the segment strings. Now, the quantity is removed from the formulaString so we will only compared the length of the segments to the length of the string without the formula quantity.